_posts/2019-07-09-10-praw-uzytecznego-designu.md

```diff
@@ -0,0 +1,124 @@
+<span class="img-legend">Rys. 1. Strona główna firmy Apple</span>
+</div>
+
+## 2. Prawo Jakoba
+
+Jakob Nielsen, jeden z czołowych specjalistów w dziedzinie użyteczności, a także założyciel Nielsen Norman Group zauważył, że użytkownicy spędzają większość czasu na innych stronach. Oznacza to, że użytkownicy wolą, aby Twoja strona działała tak samo, jak wszystkie inne witryny, które już znają. Dobrym przykładem wykorzystania tej zasady są sklepy sprzedające odzież i obuwie on-line.
+
+Jeśli strona wygląda podobnie do innych, to wtedy użytkownicy będą z łatwością umieli poruszać się po niej, skupiając się na oferowanych produktach.
+
+<div class="img-with-legend">
+<img alt="Wybrany produkt w sklepie on-line marki Kazar" src="/assets/img/posts/2019-07-09-design/2a.jpg" />
+<span class="img-legend">Rys. 2. Wybrany produkt w sklepie on-line marki Kazar</span>
+</div>
+
+<div class="img-with-legend">
+<img alt="Wybrany produkt w sklepie on-line marki Gino Rossi" src="/assets/img/posts/2019-07-09-design/2b.jpg" />
+<span class="img-legend">Rys. 3. Wybrany produkt w sklepie on-line marki Gino Rossi</span>
+</div>
+
+
+## 3. Efekt izolacji
+
+W 1933 roku Hedwig von Restorff odkryła, że przy uczeniu się listy bezsensownych słów łatwiej zapamiętuje się te, które w jakiś sposób wyróżniają się spośród innych, na przykład są napisane większymi literami lub mają inny kolor. Jak się już pewnie domyślacie, ta zasada oprócz tego, że świetnie nadaję się jako pomoc w nauce na jutrzejszą kartkówkę z biologii, to znalazła zastosowanie również w dziedzinie użyteczności. Przyjrzyjmy się chociażby Material Design, czyli jednemu z najpopularniejszych obecnie trendów w tworzeniu interfejsów. Jednym z charakterystycznych elementów tego stylu jest umieszczony w dolnym prawym rogu ekranu okrągły przycisk. Jeśli chcemy zwrócić uwagę użytkownika na dany element strony wyróżnijmy go na tle innych obiektów.
+
+<div class="img-with-legend">
+<img alt="Floating Action Button wykonuje główną akcję w aplikacji z systemem Android" src="/assets/img/posts/2019-07-09-design/3.jpg" />
+<span class="img-legend">Rys. 4. Floating Action Button wykonuje główną akcję w aplikacji z systemem Android<br />źródło: <a href="https://material.io/design/components/buttons-floating-action-button.html">material.io</a> - dostęp: 2019-07-07</span>
+</div>
+
+## 4. Efekt Zeigarnik
+
+Mówi o tym, że ludzie pamiętają nieukończone lub przerwane zadania lepiej niż zadania zakończone. Bluma Zeigarnik przeprowadziła serię eksperymentów sprawdzających zdolność do zapamiętywania różnych zadań. Osoby biorące udział w badaniach proszono o rozwiązywanie łamigłówek. Części z nich przeszkadzano w trakcie wykonywania poleceń, a pozostałym pozwalano w spokoju skończyć. Zeigarnik zauważyła, że czynności, których dokończenie przerwano, są o 90% lepiej zapamiętywane, niż te, które można było dokończyć. Ten efekt określany jest jej nazwiskiem i zyskał szerokie zastosowanie, także w dziedzinie użyteczności.
```

Comment:
Zastanów się czy nie mógłbyś wrzucić informacji jaki zabieg i po co stosuje Udemy, tak jak to napisałeś dla Allegro?

Reply:
done